target/linux/ramips/dts/mt7621_yuncore_ax820.dts

```diff
@@ -23,10 +23,29 @@
 	leds {
 		compatible = "gpio-leds";
 
-		led_status_green: status_green {
-			label = "green:status";
+		/* At least 2 AX820 variants are known and cannot be distinguished (same hw rev):
+		   - System LED green GPIO 15; red and blue apparently not connected
+		   - System LED red GPIO 15, WLAN2G LED green GPIO 14, WLAN5G LED blue GPIO 16
+		   The canonical values taken from a Yuncore datasheet are:
+		   - System LED green GPIO 15, WLAN2G LED blue GPIO 14, WLAN5G LED red GPIO 16
+		   As a result, all 3 LEDs are defined even if only one may be available, and
+		   the color information is absent as it is not reliable */
+		wlan2g {
+			label = "dome:wlan2g";
```

Comment:
According to the common binding for LEDs, the `color` property is not required. If we ever switch to automatic labels (since `label` is deprecated upstream), this would just become "wlan2g", if that's the value provided for `function`. Any objections against just dropping the "dome:" prefix?